tools/dockerfile/test/sanity/Dockerfile

```diff
@@ -94,6 +94,9 @@ ENV CLANG_FORMAT=clang-format
 RUN ln -s /clang+llvm-5.0.0-linux-x86_64-ubuntu14.04/bin/clang-tidy /usr/local/bin/clang-tidy
 ENV CLANG_TIDY=clang-tidy
 
+# Install Bazel 0.20.0
```

Comment:
please update docker images in #17798 first and then update this PR (I guess that's the plan anyway).

Reply:
Aha, that's the plan.